src/1_utils.js

```diff
@@ -605,6 +605,19 @@ utils.base64encode = function(input) {
 	return output;
 };
 
+/**
+ * Check if a String is a BASE64 encoded value
+ * @param {string} str 
+ */
+utils.isBase64Encoded = function(str){
+	if(typeof str != "string"){ return false; }
```

Comment:
what if str is null?

Reply:
null is of type "object", or if you mean undefined - type of will return "undefined". So this function will correctly return false